docs/integrations/databases/hasura.mdx

```diff
@@ -1,5 +1,166 @@
 ---
-sanity_slug: /docs/integration/hasura
+title: Hasura
+description: Learn how to integrate Clerk into your Hasura application.
 ---
 
-# This is a stub
+# Getting started
```

Comment:
The H1 of these database pages should probably not just be "Getting Started", and it should include the name of the database in question.  I looked horizontally against other DB pages and they all seem to be starting with just "Getting started".

The reason for this is twofold:

- The first one is user readability perspective; we want to ensure the user is contextualized on the page so they know what they are looking at.  Right now the sidebar is the only thing that contextualizes the user on the page to know what DB they are looking at.  On mobile this sidebar is less prevalent hidden behind a click.
- Secondarily, there may be some SEO impacts if we have a bunch of pages with H1s that are just "Getting started"

I would look at how we approach our quick-start guides, as I think those H1s are well-formed. https://clerk.com/docs/quickstarts/nextjs

Reply:
Yep, there's a ticket to fix this once these are all merged. Because some are already merged with these H1's, this can be done all at once.
Ticket: https://linear.app/clerk/issue/DOCS-342/fix-integrations-h1s

Reply:
Cool cool; happy for this to be deferred, that won't hold up my approval then as long as we have a ticket to address!